packages/nx/src/plugins/js/lock-file/pnpm-parser.ts

```diff
@@ -159,6 +163,13 @@ export function stringifyPnpmLockfile(
   return stringifyToPnpmYaml(output);
 }
 
+function normalizeLockfileVersion(version: string | number) {
```

Comment:
Version 5.4 was stored as number or string (in case of suffixes e.g. `5.4-inlineSpecifiers`).
The v6 version is stored as string.